.github/workflows/registry-build-publish.yml

```diff
@@ -14,26 +14,34 @@ env:
     IMAGE_NAME: ${{ github.repository }}
 
 jobs:
-    test:
-        name: 🧪 Test
+    lint:
+        name: 🔍 Formatting & Linting
         runs-on: ubuntu-latest
-        strategy:
-            matrix:
-                node-version: [20.9.0]
-
         steps:
             - name: ⬇️  Checkout repository
               uses: actions/checkout@v4.2.2
             - name: 🔨 Setup node
-              uses: actions/setup-node@master
+              uses: actions/setup-node@v4.4.0
               with:
-                  node-version: ${{ matrix.node-version }}
+                  node-version-file: '.nvmrc'
             - name: 📲 Install dependencies
               run: npm ci
-            - name: Format
+            - name: Prettier
               run: npx prettier --check .
-            - name: 📝 Test
-              run: npm run test-headless
+    # test:
```

Comment:
Why keep it commented out? If we want to restore it, we can just go back in the git history. Most likely we'll never restore it and then it just clutters the file

Reply:
I like your thinking! Totally agree, removed it.